CHANGELOG.md

```diff
@@ -4,6 +4,16 @@
 
 This release adds experimental trace propagation support. When enabled, by setting `propagateTraceContext` to `true` in the `networkRequestCallback` handler, a `traceparent` header will be added to the outgoing network request.
 
+### Added
+
+- (plugin-react-native-navigation) Added `@bugsnag/plugin-react-native-navigation-performance` package
+- (plugin-react-navigation) Added `@bugsnag/plugin-react-navigation-performance` package
+
+### Deprecated
+
+- (react-native-navigation) Deprecated `@bugsnag/react-native-navigation-performance` package
+- (react-navigation) Deprecated `@bugsnag/react-navigation-performance` package
+
```

Comment:
Should these be a single `Changed` entry?

Reply:
Yes please – I think this could be better explained as a single entry to explain what we're doing as it's almost impossible to grok that from these descriptions. Basically just to spell out that people should switch any existing references to the new naming, but that they old packages still exist and can be used until the next major (when we won't make promises about their working).